mypy/fastparse.py

```diff
@@ -1180,6 +1195,29 @@ def visit_ClassDef(self, n: ast3.ClassDef) -> ClassDef:
         self.class_and_function_stack.pop()
         return cdef
 
+    def validate_type_param(self, type_param: Any) -> None:
```

Comment:
It should accept `ast_TypeParam` alias, which would be similar to `ast_TypeAlias` and others.

Reply:
Actually, there is no `ast.TypeParam` but there is an `ast.TypeVar`. I think you mean it.